pyboolnet/control_strategies.py

```diff
@@ -1,5 +1,9 @@
-from itertools import combinations, product
+# This module contains the necessary functions for the control strategy identification methods described and discussed in
```

Comment:
No need to justify imports! Especially if they are standard libraries, please remove.

Reply:
This was meant to introduce the functions in the `control_strategy.py` module, not a specific import. If this not the right place, please let me know where I could do it.

Reply:
Ah, I see. That's not necessary. Please remove. You can (should?) add a section in **docs/manual.rst**. Can be very short to start out with, of course, since writing a good manual takes a lot of time..